pkg/apis/backupdriver/v1/snapshot.go

```diff
@@ -15,6 +15,11 @@ type SnapshotSpec struct {
 	// SnapshotCancel indicates request to cancel ongoing snapshot.  SnapshotCancel can be set at anytime before
 	// the snapshot reaches a terminal phase.  If the snapshot has reached a terminal phase
 	SnapshotCancel bool `json:"snapshotCancel,omitempty"`
+
+	// Paravirt or guest snapshot namespace. This is used only for supervisor snapshots that have
+	// corresponding snapshots in the guest cluster
+	// +optional
+	PVSnapshotNamespace string `json:"pvSnapshotNamespace,omitempty"`
```

Comment:
This should be called paravirtSnapshotNamespace or gcSnapshotNamespace to avoid confusion.   PV sounds like PersistentVolume especially when it is next to Snapshot.

Also since this parameter is inside Snapshot CR, can we remove "snapshot" from it?  Will we have another namespace that is not for Snapshot?

Also is it possible to have multiple guest cluster namespaces mapped to the supervisor snapshot in the future?  Should this be an array?

Reply:
Updated to paravirtSnapshotNamespace.

The reason I say paravirtSnapshotNamespace is because paravirtNamespace could mean "velero" namespace in the guest. paravirtSnapshotNamespace points to the namespace in the guest where the corrsponding snapshot CR is created.

This need not be a array as there is a 1:1 relation between a guest and a supervisor snapshot CR.

Reply:
Removed this and added a map to keep track of all supervisor snapshot CRs in the guest.

I  also plan to add a label based filter in the informer so the listeners can only pick up supervisor objects that belong to this guest cluster.